python/enthic/compute_stats.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+"""
+====================================================================
+Compute statistics from companies's data after complete and check it
+====================================================================
+
+Coding Rules:
+
+- Snake case for variables.
+- Only argument is configuration file.
+- No output or print, just log and files.
+"""
+import math
+from enthic.arbre_compte_de_resultat import TREE_VIEW
+
+def convert_data_to_tree(raw_data):
+    """
+    Returns given data as a tree like "arbre_compte_de_resultat"
+        :param raw_data : data as enthic API returns it
+        :return: given data as a tree
+    """
+    new_tree = TREE_VIEW
+    recursive_fill_tree(new_tree, raw_data)
+    return new_tree
+
```

Comment:
PEP 8: E302 expected 2 blank lines, found 1, use autopep 8 or an IDE that reformat according to PEP8 e.g. Pycharm Ctrl+Alt+L .